android/build.gradle

```diff
@@ -1,6 +1,8 @@
 group 'com.dooboolab.fluttersound'
 version '1.0-SNAPSHOT'
 
+ext.flutterFFmpegPackage = 'audio'
```

Comment:
Could you kindly explain why this is necessary?

Reply:
Actually this is not really necessary : Android still does not use ffmpeg.
(It will probably be used later, for recording MP3, for example).

I am commenting out this line in the current version.